README.md

```diff
@@ -192,8 +192,8 @@ handler.Handle(
 
         // unmarshal the message payload into a model.Response object
         // this is a wrapper transport uses when being used as a server, 
-        // it encapsulates a rich set of data
-        // about the message, but you only really care about the payload (body)
+        // it encapsulates a rich set of data about the message, 
+    	// but you only really care about the payload (body)
```

Comment:
let's be pixel/indentation perfect? 😁

Reply:
intelliJ is really odd with markdown sometimes.